_includes/content/lb4-project-commands.html

```diff
@@ -0,0 +1,14 @@
+<table width="750">
+  <thead><tr>
+  <th width="270">Commands</th>
+  <th>Description</th>
+  <th width="200">See</th>
+  </tr></thead>
+  <tbody>
+<tr>
+<td> <code>your command</code>
+</td><td>  Description of your command
+</td><td>  <a href="Dead-link.html">Link to your command page</a>
+</td></tr>
+
+</tbody></table>
```

Comment:
Needs a trailing newline.

Reply:
Done.
For some reason, the changes don't show up on github